templates/common/declarative-agent-typespec/actions.tsp.tpl

```diff
@@ -0,0 +1,37 @@
+import "@typespec/http";
```

Comment:
Please rename this file to `actions.tsp` as there is no env variable in this file.